modules/dataframe/include/inviwo/dataframe/datastructures/dataframe.h

```diff
@@ -240,13 +240,40 @@ struct DataTraits<DataFrame> {
         tb(H("Number of Rows: "), data.getNumberOfRows());
 
         utildoc::TableBuilder tb2(doc.handle(), P::end());
-        tb2(H("Col"), H("Format"), H("Rows"), H("Name"));
+        tb2(H("Col"), H("Format"), H("Rows"), H("Name"), H("Min"), H("Max"));
         // abbreviate list of columns if there are more than 20
         const size_t ncols = (data.getNumberOfColumns() > 20) ? 10 : data.getNumberOfColumns();
 
+        std::string minString{};
+        std::string maxString{};
+
         for (size_t i = 0; i < ncols; i++) {
+            auto col = data.getColumn(i);
+            col->getBuffer()
+                ->getRepresentation<BufferRAM>()
+                ->dispatch<void, dispatching::filter::Scalars>(
+                    [&minString, &maxString](auto brprecision) {
+                        const auto &vec = brprecision->getDataContainer();
+
+                        if (vec.empty()) {
+                            minString = maxString = '-';
+                            return;
+                        }
+
+                        auto [min, max] = std::minmax_element(vec.begin(), vec.end());
+                        minString = std::to_string(*min);
+                        maxString = std::to_string(*max);
+
+                        using ValueType = util::PrecisionValueType<decltype(brprecision)>;
+
+                        if constexpr (std::is_floating_point_v<ValueType>) {
+                            minString = minString.substr(0, minString.rfind('.') + 3);
+                            maxString = maxString.substr(0, maxString.rfind('.') + 3);
```

Comment:
I think this will have problem if thata contains `nan` or `inf`

Reply:
Could the existing algorithms be used (is it possible to move the info generation to base):
https://github.com/inviwo/inviwo/blob/master/modules/base/include/modules/base/algorithm/dataminmax.h

Reply:
I think the use of the dataminmax in base is smart. But it should still be done here, not moved to base. 
the `util::bufferMinMax` reaturn dvec4 even for scalars and vec2/3, so some care still need to be taken into acount when converting to string.

Reply:
> 
> 
> I think this will have problem if thata contains `nan` or `inf`

You can use `fmt::format("{:.3}", 3.14)` to format all floats to have a maximum of 3 decimals

Reply:
and what about data being in the range of -0.00000041 and 0.000000003? -> `min = 0.000, max = 0.000`?

Reply:
We could use std::scientific for floating point numbers?